src/summary/level-two-routes.component.tsx

```diff
@@ -3,6 +3,7 @@ import { match, Switch, Route } from "react-router";
 import { AllergyCardLevelTwo } from "./history/allergy-card-level-two.component";
 import { getCurrentPatient } from "@openmrs/esm-api";
 import DimensionsCardLevelTwo from "./documentation/dimensions-card-level-two.component";
+import PatientNotes from "./history/patient-notes/patient-notes-card-level-2.component";
```

Comment:
It is easier to read React code that names imported variables the same way as the they are named inside of the component

```suggestion
import PatientNotesCardLevel2 from "./history/patient-notes/patient-notes-card-level-2.component";
```